tests/services/schemas/test_creator_contributor.py

```diff
@@ -94,20 +107,6 @@ def test_creator_organization_valid_full():
 
 
 def test_creatibutor_name_edge_cases():
-    # Pass in name and given_name: name is ignored
```

Comment:
moderate: does the logic of names change when removing the requirement of `given name`? I cannot find in the issue description. If not, this test should not be removed, it should pass :) 

The logic for this is present at https://github.com/inveniosoftware/invenio-rdm-records/pull/567/files#diff-a0d8874bbf9ded2c080996a386a79aa7cc118b7054da68b2281ed5153830b419L99

Reply:
The test should indeed not be remove, just changed a bit. ``given_name`` should be changed to ``family_name`` in both valid and expected cases, and then it should pass.